pkg/operation/botanist/component/resourcemanager/resource_manager.go

```diff
@@ -1747,6 +1758,47 @@ func GetHighAvailabilityConfigMutatingWebhook(namespaceSelector, objectSelector
 	}
 }
 
+// GetEndpointSliceHintsMutatingWebhook returns the endpoint-slice-hints mutating webhook for the resourcemanager component for reuse
```

Comment:
I used `endpoint-slice-hints mutating webhook` to keep it similar with the other `Get{Name}Webhook` funcs:
https://github.com/gardener/gardener/blob/34f4e3ba443884a7ba7a3ccb3b938be3c620f40c/pkg/operation/botanist/component/resourcemanager/resource_manager.go#L1694-L1695
https://github.com/gardener/gardener/blob/34f4e3ba443884a7ba7a3ccb3b938be3c620f40c/pkg/operation/botanist/component/resourcemanager/resource_manager.go#L1651-L1652
https://github.com/gardener/gardener/blob/34f4e3ba443884a7ba7a3ccb3b938be3c620f40c/pkg/operation/botanist/component/resourcemanager/resource_manager.go#L1605-L1606

They have the webhook name prefix used in the doc string of the func.

Reply:
This particular doc string is fine. What I meant is that at some places `endpoint-slice-hints mutating webhook` is used, like here, and in some others `EndpointSlice hints webhook`, example `pkg/apis/resources/v1alpha1/types.go`:

```
	// EndpointSliceHintsConsider is a constant for a label on an Service which indicates that the EndpointSlices of the
	// Service should be considered by the EndpointSlice hints webhook. This label is added to the Service object, Kubernetes
	// maintains the Service label as EndpointSlice label. Finally, the EndpointSlice hints webhook mutates EndpointSlice resources
	// containing this label.
```

Reply:
I changed the doc string of the `GetEndpointSliceHintsMutatingWebhook` func to use `EndpointSlice hints mutating webhook` instead of `endpoint-slice-hints mutating webhook`. The former is what was used in other doc strings and in the docs.